CMakeLists.txt

```diff
@@ -112,20 +118,28 @@ endif()
 
 # Special treatment for SYCL
 if ("${YAKL_ARCH}" STREQUAL "SYCL")
-  if(NOT MKL_ROOT)
-    set(MKL_ROOT $ENV{MKLROOT} CACHE PATH "Folder contains MKL")
-  endif(NOT MKL_ROOT)
-  message("-- ** Looking for MKL installation, ${MKL_ROOT}")
-  add_compile_options(-fsycl-default-sub-group-size=16)
-  add_compile_definitions(MKL_ILP64)
-  find_path(MKL_INCLUDE mkl.h HINTS $ENV{MKLROOT} ${MKL_ROOT} PATH_SUFFIXES include)
-  target_include_directories(${YAKL_TARGET} INTERFACE "${MKL_INCLUDE}")
-  set(CMAKE_FIND_LIBRARY_SUFFIXES ".a")
-  find_library(MKL_SEQU NAMES mkl_sequential PATHS ${MKL_ROOT}/lib/intel64 REQUIRED)
-  find_library(MKL_CORE NAMES mkl_core PATHS ${MKL_ROOT}/lib/intel64 REQUIRED)
-  find_library(MKL_SYCL NAMES mkl_sycl PATHS ${MKL_ROOT}/lib/intel64 REQUIRED)
-  find_library(MKL_IPL64 NAMES mkl_intel_ilp64 PATHS ${MKL_ROOT}/lib/intel64 REQUIRED)
-  target_link_libraries(${YAKL_TARGET} INTERFACE "${YAKL_COMPILER_FLAGS} ${MKL_SYCL} -Wl,-export-dynamic -Wl,--start-group ${MKL_SEQU} ${MKL_IPL64} ${MKL_CORE} -Wl,--end-group -lsycl -lOpenCL -lpthread -lm -ldl")
+  if (YAKL_SYCL_BBFFT)
+    message("-- Building with BBFFT")
+    set(CMAKE_PREFIX_PATH ${YAKL_SYCL_BBFFT_HOME})
+    find_package(bbfft-sycl REQUIRED)
```

Comment:
This overrides CMAKE_PREFIX_PATH, but CMAKE_PREFIX_PATH is intended to be set externally by the user. Instead of these two lines, I'd suggest
```
find_package(bbfft-sycl REQUIRED HINTS ${YAKL_SYCL_BBFFT_HOME})
```